### ISSUE TITLE
Notes are pushed to "Conflicts" in a joplin profile in which sync is disabled.

### ISSUE THREAD

**Issue by poopie-head** [laurent22/joplin#1643]
### Operating system

- Windows

### Application

- Desktop

***
## Sync status (synced items / total items)

Note: 41/7
Folder: 11/4
Resource: 24/24
Tag: 7/7
NoteTag: 7/7
MasterKey: 0/0
Revision: 54/54
Total: 144/103
 
Conflicted: 1
To delete: 0
Folders
Welcome! (Desktop): 4 notes
Markdown: 1 notes
2019 BuJo: 1 notes
Conventions/Format: 1 notes
***

This is a particular "instance" of Joplin where I use a different profile I launch via a .bat file in a different directory. 
On editing a note it is immediately pushed into Conflicts. Synchronisation is disabled as i only use this set on notes on one device.
I don't know why this is happening, And the only way i can resolve it is my manually dragging the note into the right notebook. At some point I think I'm going to drag a note into the wrong notebook.

[syncReport-1560074361927.zip](https://github.com/laurent22/joplin/files/3269405/syncReport-1560074361927.zip)

_Originally posted by @poopie-head in https://github.com/laurent22/joplin/issues/70#issuecomment-500200894_

**Comment by poopie-head** [comment]
Hmm... 
![Properties](https://user-images.githubusercontent.com/44089509/59211461-85271b80-8bcd-11e9-9c0b-911938c13fe8.PNG)

So, I made a change to the shortcut properties. The "Start in" field value was set to the same as the profile directory.

This seems to have solved that issue of "Conflicts" appearing immediately after an edit after launching with a profile(in `cmd.exe` , with the command `Joplin.exe --profile "path\to\profile"`) other than the default.

Haven't experienced it since, So the issue was when launching the bat file it used to launch from the current directory and create temp files in the directory, leading to some issue when trying to match files with those in the profile directory.

**Comment by poopie-head** [comment]
Lost an entire note today. It was the note I was actively editing.
There's no trace of it. I did search through the database too, the row associated with the note was empty. No setting were changed. This profile does not sync to any target.
And neither did it turn up in the Conflicts or Deleted notebooks.
What I did notice was that no .md files associated with the notes were being generated/stored in the directory from where Joplin was being launched. Maybe that should have been enough of a warning.
Maybe this particular way of running Joplin, with files in a container, isn't currently a good way of securing a local copy of one's notes.
Is there any way of making this work?
Joplin in a VeraCrypt container, specifically.
